ansible/playbooks/jenkins/worker/fix.yml

```diff
@@ -0,0 +1,60 @@
+---
+
+#
+# Fix common problems with Jenkins workers
+#
```

Comment:
Can you add a note here making it explicit that this cannot be run while Jenkins is active? Something like:
```
Please make sure no jobs are running on this worker before starting this
script. The worker must be marked as offline in Jenkins, or by any other
means it must not be possible to start a new job on this worker while this
script is running.
```